Update dsintro.rst

yoonghm opened the pull request:
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] Added examples to create DataFrame from series and series with `nan`

yoonghm committed:
Update dsintro.rst

yoonghm commented:
For your review.
There should another method to create DataFrame from numpy series which has no index.

jorisvandenbossche commented on doc/source/dsintro.rst:
```diff
@@ -366,13 +366,32 @@ dictionary.
 
-From a Series
-~~~~~~~~~~~~~
+From a Panda Series
```
Panda -> pandas

jorisvandenbossche commented on doc/source/dsintro.rst:
```diff
@@ -366,13 +366,32 @@ dictionary.
 
+.. ipython:: python
+
+   df = pd.DataFrame()
```
We typically don't want to encourage creating an empty dataframe first, and then filling it. 

You can give an example of directly creating a DataFrame from a Series like:

```
s = pd.Series([1, 2, 3])
pd.DataFrame(s)
```

jorisvandenbossche commented on doc/source/dsintro.rst:
```diff
@@ -366,13 +366,32 @@ dictionary.
+   df
+   x2 = pd.Series([5,5], index=[1, 3])
+   x2
+   df['B'] = x2
```
This is more an example of how to add a column to existing dataframe, and how the values get aligned first before getting added. 
So I think it is not fully fitting for here (this section is only about creating a DataFrame)

yoonghm replied:
Okay, I will update that.

yoonghm replied:
Is there a place I could document
- Adding/replacing a column from a numpy series with more than 1 columns
- Adding/replacing multiple columns from a numpy series with multiple columns.

Please reject the pull request.

jorisvandenbossche replied:
There is already a section in this file about adding columns: http://pandas.pydata.org/pandas-docs/stable/dsintro.html#column-selection-addition-deletion

yoonghm committed:
Create DataFrame from numpy series

Added examples to create DataFrame from numpy series, with same and extra number of columns.

yoonghm committed:
Update dsintro.rst

yoonghm commented:
Hi,

  x2 is a ndarrays of 5 x 5. When I assign it to df['B'], only the first
column of x2 is included and there is warning whatsoever. Is this feature
documented?

On Fri, Sep 21, 2018 at 5:18 PM Joris Van den Bossche <
notifications@github.com> wrote:

> *@jorisvandenbossche* commented on this pull request.
> ------------------------------
>
> In doc/source/dsintro.rst
> <https://github.com/pandas-dev/pandas/pull/22781#discussion_r219434319>:
>
> >
>  The result will be a DataFrame with the same index as the input Series, and
>  with one column whose name is the original name of the Series (only if no other
>  column name provided).
>
> +.. ipython:: python
> +
> +   df = pd.DataFrame()
> +   df
> +   x1 = pd.Series([1,2,3,4], index=[1,2,3,4])
> +   x1
> +   df['A'] = x1
> +   df
> +   x2 = pd.Series([5,5], index=[1, 3])
> +   x2
> +   df['B'] = x2
>
> There is already a section in this file about adding columns:
> http://pandas.pydata.org/pandas-docs/stable/dsintro.html#column-selection-addition-deletion
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/pandas-dev/pandas/pull/22781#discussion_r219434319>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEBUtyahcaXZeND2eePntWLOP7yOOXH9ks5udK7jgaJpZM4Wx_rc>
> .
>

jreback commented on doc/source/dsintro.rst:
```diff
@@ -366,13 +366,56 @@ dictionary.
+    x1
+    x2 = np.random.randint(-1.0, 10, (5))
+    x2
+    df1 = pd.DataFrame(x1, columns=['A'])
```
yeah i would rather have these examples in the doc-string of the Series/DataFrame constructor than here
